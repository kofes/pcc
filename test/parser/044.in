var
	arr : array of integer = (1,2,3);
begin
end
