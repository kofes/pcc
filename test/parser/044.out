Unexpected token '=' in pos (2, 26); expected '';'';
